### PR TITLE
[SG-41353]: client/browser: migration to graphql-operations

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description
Migrate typing using from `schema.ts` to `graphql-oparations`

## Success criteria

1. All imports from `schema.ts` are replaced with respective imports from `graphql-operations`.
2. The `schema.ts` and related gulp and npm scripts are removed from the repo.

## Ref
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/41353)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-41353)

## Test Plan
- Verify that all usage of `schema.ts` types has been migrated to their equivalent typing from `graphql-oparations.ts` file in `client/browser` package.

## App preview:

- [Web](https://sg-web-contractors-sg-41353.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-kwypbupxqe.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/browser/src/shared/platform/settings.ts [hunk 1]
```diff
@@ -3,10 +3,12 @@ import { from, fromEvent, Observable } from 'rxjs'
...
 
 import { isErrorLike } from '@sourcegraph/common'
-import { dataOrThrowErrors, gql } from '@sourcegraph/http-client'
+import { dataOrThrowErrors } from '@sourcegraph/http-client'
 import { SettingsEdit } from '@sourcegraph/shared/src/api/client/services/settings'
+import { viewerSettingsQuery } from '@sourcegraph/shared/src/backend/settings'
+import { ViewerSettingsResult } from '@sourcegraph/shared/src/graphql-operations'
 import { PlatformContext } from '@sourcegraph/shared/src/platform/context'
-import * as GQL from '@sourcegraph/shared/src/schema'
+import { ISettingsCascade } from '@sourcegraph/shared/src/schema'
```
Comment (valerybugakov) [inline client/browser/src/shared/platform/settings.ts:11]: Similar note here, we still use `@sourcegraph/shared/src/schema` despite having a test plan in the PR description: 

> Verify that all usage of `schema.ts` types has been migrated to their equivalent typing from `graphql-oparations.ts` file in `client/browser` package.

Could you share reasons for keeping this import?

Comment (phunguyenmurcul) [reply client/browser/src/shared/platform/settings.ts:11]: @valerybugakov we sync the same usage of `viewerSettingsQuery` in vscode package with `ISettingsCascade` for now (both packages are using  ISettingsCascade) and intend to remove it completely from both packages in 2nd phrase of the root issue.

Comment (gitstart-sourcegraph) [reply client/browser/src/shared/platform/settings.ts:11]: @valerybugakov we sync the same usage of `viewerSettingsQuery` in vscode package with `ISettingsCascade` for now (both packages are using  ISettingsCascade) and intend to remove it completely from both packages in 2nd phrase of the root issue.

File: client/browser/src/shared/platform/context.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ export function createPlatformContext(
                     console.warn(
                         `Could not fetch Sourcegraph settings from ${sourcegraphURL} because user is not signed into Sourcegraph`
                     )
-                    updatedViewerSettings.next({ final: '{}', subjects: [] })
+                    updatedViewerSettings.next(null)
```
[the remark this answers is not in the document]
Comment (gitstart-sourcegraph) [reply client/browser/src/shared/platform/context.ts:90]: @valerybugakov we want to use the new query to remove the deprecated query. we are going to revert and push typing fixes soon

[the remark this answers is not in the document]
Comment (valerybugakov) [reply client/browser/src/shared/platform/context.ts]: Please re-request the review when it's ready 👍

File: client/browser/src/shared/platform/settings.ts (hunk 1 above)
Comment (valerybugakov) [reply client/browser/src/shared/platform/settings.ts]: Got it, makes sense!

**Review by valerybugakov — APPROVED** [review]

**Pull request merged**